source/distributed/tria.cc

```diff
@@ -3262,21 +3338,26 @@ namespace parallel
                   "Fatal Error: maximum refinement level of p4est reached."));
             }
         }
-
-      this->prepare_coarsening_and_refinement();
+   
 
       // signal that refinement is going to happen
```

Comment:
this comment belongs to the function call below, so move it there. Instead, can you add a one sentence summary of what the block of code does?